src/inspector/models/abstractinspectormodel.h

```diff
@@ -120,6 +120,7 @@ class AbstractInspectorModel : public QObject, public async::Asyncable
         TYPE_DYNAMIC,
         TYPE_TUPLET,
         TYPE_TEXT_LINE,
+        TYPE_TEMPO_RANGED_CHANGE,
```

Comment:
dito
Also TEMPO_RANGED_CHANGE_SEGMENT is missing here?

Reply:
Nope, this is a different enum with a different meaning (`InspectorModelType`)